src/Google.Maps/StaticMaps/StaticMapService.cs

```diff
@@ -77,11 +78,9 @@ public int GetMapToStream(StaticMapRequest mapOptions, System.IO.Stream outputSt
 
 			int totalBytes = 0;
 
-			WebRequest request = WebRequest.Create(requestUri);
-
-			using(WebResponse response = request.GetResponse())
+			using(HttpClient client = new HttpClient())
 			{
-				Stream inputStream = response.GetResponseStream();
+                Stream inputStream = client.GetStreamAsync(requestUri).Result;

```

Comment:
I believe .Result blocks until the value is ready just like calling the synchronous version. It would probably be better to make this async anyway, but that involved changing many more methods up the chain to support it.